helix-view/src/editor.rs

```diff
@@ -332,8 +332,6 @@ pub enum GutterType {
     Diagnostics,
     /// Show line numbers
     LineNumbers,
-    /// Show one blank space
```

Comment:
Please keep the element because it's still useful if you want to add padding between diagnostics and line number for example.

Reply:
If I add it back then it no longer fix the issue, I think probably a separate fix is needed, but for now I think it is fine.